### PR TITLE
DreamLabNetwork added to Webdeployments

### PR DESCRIPTION
**Pull request by Bangalisch**
Yo, as per #1094 i've added a page for DreamLab Network.

I might have to go over it again. But this is initial page.
Included are images in the right folder.

### CODE REVIEW

**Comment by Bangalisch** [comment]
Not included yet, but have done, is taken video footage of all the steps. 
Will work out with @webworthy how and when this can be put together for intro video :)

File: docs/Deployment/DreamlabNetwork.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
+# DreamLab Network BTCPay Server web deployment.
```
Comment (pavlenex) [inline docs/Deployment/DreamlabNetwork.md:1]:
```suggestion
# DreamLab Network BTCPay Server web deployment.
```

File: docs/Deployment/DreamlabNetwork.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
+# DreamLab Network BTCPay Server web deployment. 
+
+In this guide, we will talk you through the initial setup of your DreamLab Network BTCPay Server web deployment.
```
Comment (pavlenex) [inline docs/Deployment/DreamlabNetwork.md:3]:
```suggestion
In this guide, we will walk you through the initial setup of your DreamLab Network BTCPay Server web deployment. 
```

File: docs/Deployment/DreamlabNetwork.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
+# DreamLab Network BTCPay Server web deployment. 
+
+In this guide, we will talk you through the initial setup of your DreamLab Network BTCPay Server web deployment. 
+DreamLab offers a variety of services. From domains, Dedicated game servers, VPS servers, and now BTCPay Server as a service.
```
Comment (pavlenex) [inline docs/Deployment/DreamlabNetwork.md:4]:
```suggestion
DreamLab offers a variety of services. From domains, dedicated game servers, VPS servers, and now BTCPay Server as a service. 
```

File: docs/Deployment/DreamlabNetwork.md [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+DreamLab offers a variety of services. From domains, Dedicated game servers, VPS servers, and now BTCPay Server as a service. 
+They've built multiple plans and configurations for BTCPay Server, hosted in North America, with expansion to other locations on their premise. 
+
+## 1. Pick the server that fits your needs.
+
+On the DreamLab website, in the top menu, go to `Services` and click on `BTCPay Servers`.
+
+![DreamLab Network signup](../img/Dreamlab/btcpaydreamlab8.jpg "DreamLab Network signup")
+
+Now you are on the right page to pick a plan suited to your BTCPay Server needs.
```
Comment (pavlenex) [inline docs/Deployment/DreamlabNetwork.md:13]:
```suggestion
Next, pick a plan suited to your BTCPay Server needs. 
```

File: docs/Deployment/DreamlabNetwork.md [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+On the DreamLab website, in the top menu, go to `Services` and click on `BTCPay Servers`.
+
+![DreamLab Network signup](../img/Dreamlab/btcpaydreamlab8.jpg "DreamLab Network signup")
+
+Now you are on the right page to pick a plan suited to your BTCPay Server needs. 
+They offer three different plans.  
+
+* Basic plan 
+* Advanced plan 
+* Shared Hosted
```
Comment (pavlenex) [inline docs/Deployment/DreamlabNetwork.md:18]:
```suggestion
* Basic 
* Advanced 
* Shared 
```

File: docs/Deployment/DreamlabNetwork.md [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+
+Now you are on the right page to pick a plan suited to your BTCPay Server needs. 
+They offer three different plans.  
+
+* Basic plan 
+* Advanced plan 
+* Shared Hosted 
+
+They've enlisted what the difference between every plan is, so we won't list those out. 
+That said, we do believe that going with the `Basic plan` should be sufficient for a starting store.
```
Comment (pavlenex) [inline docs/Deployment/DreamlabNetwork.md:21]:
```suggestion
That said, we do believe that going with the `Basic plan` should be sufficient for starting a store. 
```

File: docs/Deployment/DreamlabNetwork.md [hunk 7]
```diff
@@ -0,0 +1,119 @@
...
+![DreamLab Network Plan](../img/Dreamlab/btcpaydreamlab9.jpg "DreamLab Network plan")
+
+## 2. Configure your BTCPay Server
+
+First, off they show you the basic setup of your node. 
+All the technical specs are listed. 
+
+![DreamLab Network techspecs](../img/Dreamlab/btcpaydreamlab11.jpg "DreamLab Network specs")
+
+Choose the Billing cycle that fits you.
```
Comment (pavlenex) [inline docs/Deployment/DreamlabNetwork.md:33]:
```suggestion
Choose the billing cycle that fits you. 
```

File: docs/Deployment/DreamlabNetwork.md [hunk 8]
```diff
@@ -0,0 +1,119 @@
...
+![DreamLab Network techspecs](../img/Dreamlab/btcpaydreamlab11.jpg "DreamLab Network specs")
+
+Choose the Billing cycle that fits you. 
+You can set these by 
+
+* Monthly 
+* quarterly 
+* Semi-Annually 
+* Annually 
+
```
Comment (pavlenex) [inline docs/Deployment/DreamlabNetwork.md:40]: Not really relevant imo, let's keep it concise.
```suggestion
```

File: docs/Deployment/DreamlabNetwork.md [hunk 9]
```diff
@@ -0,0 +1,119 @@
...
+Now we really go into the configuration of your server.
```
Comment (pavlenex) [inline docs/Deployment/DreamlabNetwork.md:41]:
```suggestion
Next, configure your server.
```

File: docs/Deployment/DreamlabNetwork.md [hunk 10]
```diff
@@ -0,0 +1,119 @@
...
+### Hostname 
+
+This is where you set the domain name. You can leave this as-is and use the generated address or provide it with a domain of your own. 
+Don't forget that if you set your domain, you need to create an A record that points to your VPS IP address. 
+
+![DreamLab Network domain](../img/Dreamlab/btcpaydreamlab12.jpg "DreamLab Network domain")
+
+### Configurable Options
+
+Next up, Configurable options. Here is where we set the BTCPay Server configuration. You get to set how and what you want your node to run.
```
Comment (pavlenex) [inline docs/Deployment/DreamlabNetwork.md:52]:
```suggestion
Next up, `configurable options`. Here is where you set the BTCPay Server configuration. You get to set how and what you want your node to run.
```

File: docs/Deployment/DreamlabNetwork.md [hunk 11]
```diff
@@ -0,0 +1,119 @@
...
+![DreamLab Network setup](../img/Dreamlab/configureserver.png "DreamLab Network setup")
+
+
+### Addition Information 
+
+Here you have the option to set an operating system. 
+This, as of writing, is locked to ubuntu 20.04 x86. 
+And for the ISO Disk option, there's none to take. 
+
+Click Continue to go to the payment page.
```
Comment (pavlenex) [inline docs/Deployment/DreamlabNetwork.md:73]:
```suggestion
Click `Continue` to go to the payment page. 
```

**Review by pavlenex — COMMENTED** [review]
Some nits, thanks for quickly pushing this one up, impressive. 🚀

**Pull request merged**